app/services/the_tvdb_service.rb

```diff
@@ -0,0 +1,122 @@
+class TheTvdbService
+  BASE_URL = 'https://api.thetvdb.com'.freeze
+  API_KEY = ENV['THE_TVDB_API_KEY'].freeze
+
+  attr_accessor :media_id
+
+  def initialize
+    @token = api_token
+  end
+
+  # This will update all mappings to have a season number
+  # This should be run ONLY ONCE, unless we import data from somewhere else.
```

Comment:
Is it *safe* to run multiple times?

Reply:
yup, I ran it twice in a row and causes no issues. It is more for documentation because there won't be a cron-job for this method.

Reply:
Alright